### DISCUSSION TITLE
deleting a page or page range

### DISCUSSION THREAD

**Discussion by chapmanwilliam** [pymupdf/PyMuPDF#817] [General]
Dear Jorj,

I get this error when attempting to delete a page:

 File "/Users/William/Dropbox (Personal)/Python/Code/pyCharm/pythonProject2/venv/lib/python3.9/site-packages/fitz/fitz.py", line 4882, in deletePage
    xref = self.outline_xref(i)
AttributeError: 'Document' object has no attribute 'outline_xref'

Any ideas? It used to work fine but the attribute says its gone missing.

Best wishes, William

**Comment by JorjMcKie** [comment]
Hi William,
I am sorry, this is a bug I introduced in the recent change 🙄. Forgot to rename something in course of those TOC changes.
I will open a bug in the issues.
For a hot fix you can do that rename yourself by changing `fitz.py`. Here is a difference file. Hope you know how to read it.
[diff.txt](https://github.com/pymupdf/PyMuPDF/files/5792350/diff.txt)

**Reply by JorjMcKie** [reply]
I have generated new wheels in https://github.com/pymupdf/PyMuPDF-wheels if you want to reinstall ...
